packages/lumx-react/src/components/date-picker/DatePicker.tsx

```diff
@@ -27,7 +27,7 @@ interface DatePickerProps extends GenericProps {
     todayOrSelectedDateRef?: RefObject<HTMLButtonElement>;
 
     /** Value. */
-    value: moment.Moment | undefined;
+    value: Date | moment.Moment | undefined;
```

Comment:
Should also accept a compatible string.
Compatible string format is documented here: https://momentjscom.readthedocs.io/en/latest/moment/01-parsing/02-string/